controllers/attester_controller_test.go

```diff
@@ -68,21 +68,31 @@ var _ = Context("attester controller", func() {
 		})
 
 		It("should have created a secret", func() {
-			att := rodev1alpha1.Attester{}
+			Eventually(func() rodev1alpha1.ConditionStatus {
+				att := rodev1alpha1.Attester{}
 
-			err := k8sClient.Get(ctx, types.NamespacedName{
-				Name:      attesterName,
-				Namespace: namespace.Name,
-			}, &att)
+				err := k8sClient.Get(ctx, types.NamespacedName{
+					Name:      attesterName,
+					Namespace: namespace.Name,
+				}, &att)
 
-			Expect(err).ToNot(HaveOccurred(), "error getting test attest", err)
+				Expect(err).ToNot(HaveOccurred(), "error getting test attester", err)
 
-			status := rodev1alpha1.GetConditionStatus(&att, rodev1alpha1.ConditionSecret)
-			Expect(status).To(BeEquivalentTo(rodev1alpha1.ConditionStatusTrue))
+				if err != nil {
+					return rodev1alpha1.ConditionStatusFalse
+				}
+
+				for _, cond := range att.Status.Conditions {
```

Comment:
Is there a reason to iterate over the conditions instead of using rodev1alpha1.GetConditionStatus()? I think rodev1alpha1.GetConditionStatus() has the same loop in its code.

Reply:
Good point, I took out the loop and made the test utilize that function